src/features/transfer/TransferTokenForm.tsx

```diff
@@ -235,12 +251,19 @@ function useSelfTokenBalance(tokenRoutes) {
     : route.baseChainId === originChainId
     ? tokenAddress
     : route.originTokenAddress;
-  return useAccountTokenBalance(originChainId, addressForBalance);
+  const decimals = ! route
```

Comment:
Seems like this is a boolean value right (because of the `!` char)? rename to `hasDecimals` or something more clear

Reply:
@jmrossy 
no
```
const decimals = !route ? STANDARD_TOKEN_DECIMALS : route.decimals;
```
it like your code 
```
const addressForBalance = !route
    ? ''
    : route.baseChainId === originChainId
    ? tokenAddress
    : route.originTokenAddress;
```
i want to validate route object to get route.decimals

Reply:
Got it, thanks for clarifying